cypress/integration/ambianic-tests/navbar.spec.js

```diff
@@ -1,23 +1,26 @@
 /// <reference types="cypress" />
 
+export const checkViewPort = (cy, device) => {
+  cy.viewport(device)
+  expect(cy.get("[data-cy=login]")).to.exist
+}
+
 context('Check Navbar Items', () => {
   before(() => {
     cy.visit('http://localhost:8080')
     cy.get('[data-cy=timeline]').click()
   })
 
+  it('Should have a search bar', () => {
+    cy.get('[data-cy=container').find('#searchbar')
+  })
+
   it('Should be a download off button', () => {
     const t = cy.get('[data-cy=download-off]')
     expect(t).to.exist
   })
 
-
-  /** future buttons
```

Comment:
@vickywane why did you put back on the buttons that were disabled due to lack of value at this time?